src/system_database.ts

```diff
@@ -1037,7 +1037,8 @@ export class PostgresSystemDatabase implements SystemDatabase {
         const res = await trx<workflow_status>(`${DBOSExecutor.systemDBSchemaName}.workflow_status`)
           .where('workflow_uuid', id)
           .andWhere('status', StatusString.ENQUEUED)
-          .update('status', StatusString.PENDING);
+          .update('status', StatusString.PENDING)
+          .onConflict().ignore();
```

Comment:
Hum.  In trying to get the python version, it appears that this is not documented.  Empirically it works, but update is not claimed to support onConflict and onConflict is supposed to take the constraint columns...  And PG does not have anything in its UPDATE syntax for this either.  And the SQL generated is the same... something else is going on.